### ISSUE TITLE
Didn't you apply any learning rate scheduler?

### ISSUE THREAD

**Issue by dlwogns0128** [Rudrabha/LipGAN#34]
Hello, I have a question about training LipGAN.
In the train.py code, you can set the lr to 1e-3 but there is no scheduler code to reduce learning rate.
Is it right to set learning rate to 1e-3 constantly?

**Comment by prajwalkr** [comment]
We have not experimented extensively with the learning rate schedule. The publicly released model is trained for a couple of more epochs with a lower learning rate, but we did not see a significant improvement in spite of this change. We suppose a learning rate schedule might make a significant difference if we are training on a much larger dataset. The LRS2 train set is only about 29 hours.
